makefile

```diff
@@ -44,7 +44,10 @@ lint: lint-python lint-shell
 typecheck: venv
 	$(py_env)/bin/mypy --ignore-missing-imports $(py_packages)
 
-ci: tests lint typecheck
+bandit: venv
+	$(py_env)/bin/bandit -r . -x venv/
```

Comment:
Same comment as on the other PR: Should this be looking at `$(py_env)` so that the path to the virtualenv isn't hard-coded? E.g. on Travis the virtual environment will be in a different location (see [ci.sh](https://github.com/ascoderu/opwen-cloudserver/blob/449ee96d1853996983a4b9570c13bbce52ca7ebb/travis/ci.sh#L9) where we override the py_env used by make).